packages/binding-opcua/src/opcua-client.ts

```diff
@@ -156,15 +168,15 @@ export default class OpcuaClient implements ProtocolClient {
         return { type: contentType, body: Readable.from(Buffer.from(body)) };
     }
 
-    public async writeResource(form: OpcuaForm, content: Content): Promise<any> {
+    public async writeResource(form: OpcuaForm, content: Content): Promise<void> {
         const body = await ProtocolHelpers.readStreamFully(content.body);
-        const payload: any = content ? JSON.parse(body.toString()) : {};
+        const payload: Record<string, unknown> = content ? JSON.parse(body.toString()) : {};
         const url = new Url(form.href);
         const endpointUrl = `${url.protocol}//${url.host}`;
-        const method = form["opc:method"] ? form["opc:method"] : "WRITE";
-        const contentType = "application/x.opcua-binary";
+        // const method = form["opc:method"] ? form["opc:method"] : "WRITE";
+        // const contentType = "application/x.opcua-binary";
```

Comment:
Same as above, I prefer not having code hanging around. Unless you have a good reason for it.